src/pages/HomePage/Componets/OurCustomers/CustomerLogo/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import "./CustomerLogo.css";
+
+const CustomerLogo = ({ logo: { img } }) => {
+	return (
+		<div className="customer-logo__container">
+			<div className="customer-logo">
```

Comment:
This could be a link to customers websites btw. Hard to tell from the design tho